src/App2.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div id="app">
+    <div>
+      <button @click="openDrawerMenu">ボタン</button>
+    </div>
+    <transition name="right">
+      <div v-if="drawerFlg" class="drawer-menu-wrapper">
+        <div class="drawer-menu">
+          <!-- ここにメニューの内容を書いていく -->
+        </div>
+      </div>
+    </transition>
+  </div>
+
+
+  <div id="app">
+    <img alt="Vue logo" src="./assets/logo2.jpg">
+    <HelloWorld msg="Welcome to Your Vue.js App"/>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "App",
+  data() {
+    return {
+      drawerFlg: false
+    };
+  },
+  methods: {
+    openDrawerMenu() {
+      this.drawerFlg = true;
```

Comment:
これではtrueにしかできないので、
```suggestion
      this.drawerFlg = !this.drawerFlg;
```
true,falseの反転はこんな感じで書いたりします